configure.ac

```diff
@@ -7497,6 +7499,22 @@ AS_VAR_IF([ac_cv_libatomic_needed], [yes],
 _RESTORE_VAR([CPPFLAGS])
 
 
+# gh-59705: Maximum length in bytes of a thread name
+case "$ac_sys_system" in
+  Linux*) PYTHREAD_NAME_MAXLEN=15;;
```

Comment:
Does it include Android? Bionic has the same limit.

Reply:
Android is identified as `ac_sys_system=Linux-android`, so this will match it.

Reply:
I added a comment to mention that this line is for Linux **and** Android.